hack/integration.sh

```diff
@@ -2,25 +2,53 @@
 
 set -e
 
+REDSKYCTL_BIN="${REDSKYCTL_BIN:=dist/redskyctl_linux_amd64/redskyctl}"
+
 echo "Upload image to KinD"
 [[ -n "${IMG}" ]] && kind load docker-image "${IMG}" --name chart-testing
 [[ -n "${SETUPTOOLS_IMG}" ]] && kind load docker-image "${SETUPTOOLS_IMG}" --name chart-testing
+
 echo "Init redskyops"
-dist/redskyctl_linux_amd64/redskyctl init --wait
-echo "Create postgres experiment"
-kustomize build hack | kubectl apply -f -
+${REDSKYCTL_BIN} init
```

Comment:
Why did you remove the `--wait` here?

Reply:
I can't seem to find the failed build, but I believe it was due to randomly getting an exit 1.
I'm assuming it's the same reason I had to add in the sleep below where the resources in Kubernetes weren't available/present by the time we got to waiting on the resource.